source/TradingPanel.cpp

```diff
@@ -253,6 +252,8 @@ bool TradingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 		{
 			if(it.first->Get("installable") >= 0. && !sellOutfits)
 				continue;
+			if(!player.OutfitIsKnown(*it.first))
+				continue;
```

Comment:
```suggestion
			if(it.first->Get("installable") >= 0. && !sellOutfits || !player.OutfitIsKnown(*it.first))
				continue;
```

Reply:
good suggestion, however I feel it should be like this:
if((it.first->Get("installable") >= 0. && !sellOutfits) || !player.OutfitIsKnown(*it.first))

objections?

Reply:
fixed